src/mplcatppuccin/__init__.py

```diff
@@ -9,6 +11,13 @@
 __author__ = "Bram de Wilde <contact@bramdewilde.com>"
 __all__ = []
 
+warning_message = """This package is deprecated, please upgrade to https://github.com/catppuccin/python (pip install catppuccin)
+Follow the README in the new repo to upgrade your code (minor edits)
```

Comment:
```suggestion
```
Same as above, we shouldn't say "minor edits". Also this line is redundant since they should be checking the new repository linked above anyway :)